help/destinations/catalog/advertising/amazon-ads.md

```diff
@@ -20,15 +25,28 @@ The Amazon Ads integration with Adobe Experience Platform provides turn-key inte
 
 To help you better understand how and when you should use the *Amazon Ads* destination, here are sample use cases that Adobe Experience Platform customers can solve by using this destination.
 
+### Use case #1
 ### Activation and targeting {#activation-and-targeting}
 
 This integration with Amazon DSP allows Amazon Ads advertisers to pass advertiser CDP audiences from Adobe Experience Platform to Amazon's DSP to create advertiser audiences for advertising targeting. Audiences may be selected within the Amazon DSP for positive targeting, as well as negative targeting (suppression).
 
+### Use case #2
```

Comment:
we can skip the ### Use case #1, ### Use case #2, and ### Use case #3 headings and just leave ### Activation and targeting {#activation-and-targeting}, ### Analytics and Measurement {#analytics-and-measurement}, and ### Advertising optimization {#advertising-optimization}